torch/csrc/jit/codegen/cuda/disjoint_set.h

```diff
@@ -17,7 +19,7 @@ namespace cuda {
 //! DisjointSet::areEqual(a,b) checks if a and b belong same class
 //!
 //! \note The template type T is assumed to be hashable
```

Comment:
We can remove this line of comment now if we are requiring user to supply a hash function.